src/common/loadingBar.js

```diff
@@ -0,0 +1,52 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import Bar from 'react-native-progress/Bar';
+import { getLoading } from '../selectors';
+import { BRAND_COLOR, NAVBAR_SIZE } from '../styles';
+import { connect } from '../react-redux';
+import type { Dispatch } from '../types';
+
+type SelectorProps = $ReadOnly<{|
+  loading: boolean,
+|}>;
+
+type Props = $ReadOnly<{|
+  color: string | number,
+  unfilledColor: string,
+
+  dispatch: Dispatch,
+  ...SelectorProps,
+|}>;
+
+class LoadingBar extends PureComponent<Props> {
```

Comment:
So, I probably should have realized this earlier, but I think we should keep the name `LoadingBanner`. That way, names like `shouldShowLoadingBanner`, which we have in various places right now will keep on making sense, and the diff for this file will show which bits of the code come from the old `LoadingBanner` file vs which ones are new.

Reply:
as you said :)
> Could you rename this file and class? It's not really a "progress" bar (since we're not measuring progress), I think "LoadingBar" would be better.

I would suggest to keep file name as LoadingBar, and  change  the `shouldShowLoadingBanner` to `shouldShowLoadingBar` everywhere.
wouldn't it be good?

Reply:
Yeah, that's probably a better name on the whole. The reason I suggested keeping it `LoadingBanner` is because that would make the diff nicer, but I do think `LoadingBar` describes it better. It'd be ideal for you to change `LoadingBanner` to use the `ProgressBar` code without renaming it in one commit, and then rename it (and the `shouldShowLoadingBanner`/etc) in a second commit.